examples/nidmm_measurement/tests/integration/test_measurement.py

```diff
@@ -0,0 +1,39 @@
+import measurement
+import nidmm
+import pytest
+from google.protobuf import any_pb2
+
+from ni_measurementlink_service._internal.stubs.ni.measurementlink.measurement.v2.measurement_service_pb2 import (
+    MeasureRequest,
+)
+from ni_measurementlink_service._internal.stubs.ni.measurementlink.measurement.v2.measurement_service_pb2_grpc import (
+    MeasurementServiceStub,
+)
+from ni_measurementlink_service._internal.stubs.ni.measurementlink.pin_map_context_pb2 import (
+    PinMapContext,
+)
+from tests.assets.nidmm_measurement_parameters_pb2 import Configurations, Outputs
+
+
+def test___measurement_service___measure___returns_outputs(
+    measurement_stub: MeasurementServiceStub, pin_map_id: str
+) -> None:
+    configurations = Configurations(
+        pin_name="Pin1",
+        measurement_type=nidmm.Function.DC_VOLTS.value,
+        range=10.0,
+        resolution_digits=5.5,
+    )
+    request = MeasureRequest(
+        pin_map_context=PinMapContext(pin_map_id=pin_map_id, sites=[0]),
+    )
+    request.configuration_parameters.Pack(configurations)
+
+    response_iterator = measurement_stub.Measure(request)
+
+    responses = list(response_iterator)
+    assert len(responses) == 1
+    outputs = Outputs.FromString(responses[0].outputs.value)
+    assert -10.0 <= outputs.measured_value <= 10.0
```

Comment:
This doesn't account for overrange. A simulated PCI-4065 returns values on the order of 10.000126.